### ISSUE TITLE
raises wrong import order for fallback idiom

### ISSUE THREAD

**Issue by dcbaker** [PyCQA/pylint#750]
```python
try:
    import simplejson as json
except ImportError:
    import json
```

Pylint (as of 1.5.1) incorrectly marks this as wrong import order.
